README.md

```diff
@@ -1,7 +1,7 @@
 # Math-Magicians
 
```

Comment:
- [ ] Technically, users can not successfully run `npm start` on your application without first installing the packages. Kindly add instructions for doing that ie **`npm install`**.

  ![rosh-npm](https://user-images.githubusercontent.com/36542986/148074066-93d8e85a-81be-4dac-a69b-528d84203711.jpg)